tests/runtime/test_file_handlers.py

```
@@ -447,6 +447,34 @@ async def test_files_read_svg_returns_base64_and_mime(fake_config, fake_provider
 
+@pytest.mark.asyncio
+async def test_files_read_svg_as_text_returns_content(fake_config, fake_provider, tmp_path):
+    """files.read on .svg + as_text=true 走文本分支（#776）。
+
+    svg 同时属文本安全集与二进制可读集，默认二进制（前端内联展示）；
+    as_text=true 时显式请求纯文本，agent 可读 svg 源码。
+    """
+    from miqi.runtime.app_server import ClientSessionRegistry
+    from miqi.runtime.file_handlers import files_read_handler
+
+    sm, ws = _setup_session("svg-text", "client-1")
```

Ruff (0.16.1)

[warning] Unpacked variable `sm` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)